pkg/plugins/models.go

```diff
@@ -21,14 +21,29 @@ var (
 	PluginStateAlpha PluginState = "alpha"
 )
 
-type PluginSignature string
+type PluginSignatureState struct {
+	Status     PluginSignatureStatus
+	Type       PluginSignatureType
+	SigningOrg string
+}
+
+type PluginSignatureStatus string
+
+const (
+	PluginSignatureInternal PluginSignatureStatus = "internal" // core plugin, no signature
+	PluginSignatureValid    PluginSignatureStatus = "valid"    // signed and accurate MANIFEST
+	PluginSignatureInvalid  PluginSignatureStatus = "invalid"  // invalid signature
+	PluginSignatureModified PluginSignatureStatus = "modified" // valid signature, but content mismatch
+	PluginSignatureUnsigned PluginSignatureStatus = "unsigned" // no MANIFEST file
+)
+
+type PluginSignatureType string
 
 const (
-	PluginSignatureInternal PluginSignature = "internal" // core plugin, no signature
-	PluginSignatureValid    PluginSignature = "valid"    // signed and accurate MANIFEST
-	PluginSignatureInvalid  PluginSignature = "invalid"  // invalid signature
-	PluginSignatureModified PluginSignature = "modified" // valid signature, but content mismatch
-	PluginSignatureUnsigned PluginSignature = "unsigned" // no MANIFEST file
+	Grafana    PluginSignatureType = "grafana"
+	Commercial PluginSignatureType = "commercial"
+	Community  PluginSignatureType = "community"
+	Private    PluginSignatureType = "private"
```

Comment:
It's better if you disambiguate these as a variable name such as `grafana` is extremely ambiguous and will easily clash with other variable names.
```suggestion
	GrafanaType    PluginSignatureType = "grafana"
	CommercialType PluginSignatureType = "commercial"
	CommunityType  PluginSignatureType = "community"
	PrivateType    PluginSignatureType = "private"
```

Furthermore, `Community` and `Commercial` are unused. Can we drop them?